DOC 423 OOTB Metrics (#504)
amplitude/amplitude-docs: content/collections/analytics/en/out-of-the-box-metrics.md

---
id: fb3c2b70-88d2-4521-8b60-6e09adf673dd
blueprint: analytic
title: 'Out-of-the-box Metrics'
landing: false
exclude_from_sitemap: false
updated_by: 0c3a318b-936a-4cbd-8fdf-771a90c297f0
updated_at: 1738269005
---

> Amplitude’s Out-of-the-Box (OOTB) metrics provide consistent, validated definitions for common performance indicators. Whether you’re tracking page views on the web, ad performance, or time spent, OOTB metrics share one synced definition across all your Amplitude projects. When you edit an OOTB metric once, it updates everywhere—saving you time, reducing errors, and aligning teams around a single source of truth.

Reviewer: - We haven't released ads or time spent metrics yet.
- Specify "within a project" in case they think metric updates go x-app

```suggestion
Amplitude’s Out-of-the-Box (OOTB) metrics provide consistent, validated definitions for common performance indicators. OOTB metrics share one synced definition across all your Amplitude projects. When you edit an OOTB metric once, it updates everywhere within a project—saving you time, reducing errors, and aligning teams around a single source of truth.
```

## Advantages of OOTB metrics

* **Consistency and alignment**: These metrics are both dynamic and synced. You don't need to worry about mismatched definitions or manual updates in more than one place.
* **Speed and scalability**: Amplitude provides default metrics, so you can start analyzing key performance indicators quickly, and keep consistent definitions across teams and projects.

## Available metrics

Amplitude provides the following metrics for web analytics, which use the `[Amplitude] Page Viewed` event:

* Visitors
* Page Views
* Bounce Rate
* Entry Rate
* Exit Rate
* Page Views per Session
* Session Entries
* Session Exits

Regardless of where you use them, for example in a [Data Table](docs/analytics/charts/data-tables), [Event Segmentation](/docs/analytics/charts/event-segmentation) chart, or OOTB Web Analytics, they reference the same underlying definitions. Updates you make to the metric definition in one place applies everywhere else you use that metric.

## Edit existing metrics

> Access OOTB metrics from any [Data Table](docs/analytics/charts/data-tables) or [Event Segmentation](/docs/analytics/charts/event-segmentation) chart. To edit an OOTB metric:

Reviewer: They can edit in the web analytic settings too (it's all synced to settings)!

```suggestion
Access OOTB metrics from a [Data Table](docs/analytics/charts/data-tables), [Event Segmentation](/docs/analytics/charts/event-segmentation) chart, or the Web Analytics settings. To edit an OOTB metric:
```
